plugins/modules/backup_selection.py

```diff
@@ -69,8 +70,71 @@
   conditions:
     description:
       - A list of conditions (expressed as a dict) that you define to assign resources to your backup plans using tags.
+      - When you specify more than one condition in I(conditions), you only assign the resources that match ALL conditions (using AND logic).
       - I(conditions) supports C(StringEquals), C(StringLike), C(StringNotEquals), and C(StringNotLike). I(list_of_tags) only supports C(StringEquals).
```

Comment:
Not sure if this phrase is accurate at this point, it should probably reference the snake cased values?